beacon/weakcoin/weak_coin.go

```diff
@@ -193,7 +192,7 @@ func (wc *WeakCoin) FinishEpoch(ctx context.Context, epoch types.EpochID) {
 	if epoch != wc.epoch {
 		logger.With().Fatal("attempted to finish beacon weak coin for the wrong epoch",
 			epoch,
-			log.Stringer("weak_coin_epoch", wc.epoch),
+			log.FieldNamed("weak_coin_epoch", wc.epoch),
```

Comment:
can you Uint32(), my idea was that we remove FieldNamed over time and then it reduces amount of refactoring